app/views/layouts/menus/_instructor_ta_sub_menu_assessments.html.erb

```diff
@@ -42,19 +42,27 @@
   <% elsif assessment.is_a?(Assignment) && !assessment.new_record? %>
     <% target_id = @result&.is_a_review? ? assessment.pr_assignment.id :
                                                      assessment.id %>
-    <% if allowed_to?(:manage?, with: AssignmentPolicy) %>
-      <li <% if ['criteria',
-                 'annotation_categories',
-                 'automated_tests',
-                 'exam_templates'].include?(controller.controller_name) ||
+    <% if allowed_to?(:manage?, with: AssignmentPolicy) ||  allowed_to?(:see_starter_files?, with: AssignmentPolicy) %>
+    <li <% if ['criteria',
```

Comment:
Looks like there was a whitespace change here that you should revert. Make sure everything here is indented within the outer `if`